modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -13,101 +16,118 @@ Contact your customer service representative about pricing for this feature.
 
-Mule runtimes support three basic types of transports:
+Mule runtime engines support three basic types of transports:
 
-* Socket-based transports read input sent to network sockets that Mule owns. Examples include TCP, UDP, and HTTP[S].
-* listener-based transports read data using a protocol that fully supports concurrent multiple accessors. Examples include JMS and VM.
-* resource-based transports read data from a resource that allows multiple concurrent accessors, but does not natively coordinate their use of the resource. For instance, suppose multiple programs are processing files in the same shared directory by reading, processing, and then deleting the files. These programs must use an explicit, application-level locking strategy to prevent the same file from being processed more than once. Examples of resource-based transports include File, FTP, SFTP, E-mail, and JDBC.
+* Socket-based transport: reads input sent to network sockets that Mule owns. +
+Examples include TCP, UDP, and HTTP[S].
+* Listener-based transport: reads data using a protocol that fully supports concurrent multiple accessors. +
+Examples include JMS and VM.
+* Resource-based transport: reads data from a resource that allows multiple concurrent accessors, but does not natively coordinate their use of the resource. +
+For instance, suppose multiple programs are processing files in the same shared directory by reading, processing, and then deleting the files. These programs must use an explicit, application-level locking strategy to prevent the same file from being processed more than once. Examples of resource-based transports include File, FTP, SFTP, E-mail, and JDBC.
 
 All three basic types of transports are supported in clusters in different ways, as described below.
 
 * Socket-based
-** Since each clustered Mule runtime runs on a different network node, each instance receives only the socket-based traffic sent to its node. Incoming socket-based traffic should be <<Clustering and Load Balancing>> to distribute it among the clustered instances.
+** Because each clustered Mule runtime engine runs on a different network node, each instance receives only the socket-based traffic sent to its node. You need to apply clustering and load balancing to distribute incoming socket-based traffic among the clustered instances.
 ** Output to socket-based transports is written to a specific host/port combination. If the host/port combination is an external host, no special considerations apply. If it is a port on the local host, consider using that port on the load balancer instead to better distribute traffic among the cluster.
 * Listener-based
 ** Listener-based transports fully support multiple readers and writers. No special considerations apply either to input or to output.
```

Comment:
Review everything that talks about transports

Reply:
Done here: 6c24446